src/test/components/Calculator.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] We have been advised by a previous reviewer that the creation of duplicate files is not necessary, as they can be tested directly. In light of this, we kindly request that all duplicate component files be removed. These files include:
```
 Calculator.js
 Navbar.js
 Home.js
 Quote.js
 Calculator.js
 Operate.js
```
 Please ensure that the necessary steps are taken to ensure that this is done in a timely and efficient manner. Thank you for your attention to this matter.

Reply:
Hello @Rustamxon7  Please consider that we did this to avoid some unexpected changes in our program.